### ISSUE TITLE
Move to File->Export

### ISSUE THREAD

**Issue by LanceJZ** [reattiva/Urho3D-Blender#72]
Is there any way to move it to the File->Export menu where it belongs?

**Comment by reattiva** [comment]
No, you have to change the code. Look at how the STL exporter ("io_mesh_stl" addon) was done.
